source/api/discovery/0.5/index.md

```diff
@@ -82,127 +82,138 @@ The key words _MUST_, _MUST NOT_, _REQUIRED_, _SHALL_, _SHALL NOT_, _SHOULD_, _S
+Activities can be added to or removed at times other than when the resources they refer to are created, modified, deleted or moved. There are several use cases for this pattern, however they are all outside of the normal publication of a stream by the content owners and thus are unlikely to affect most publishing implementations. They do affect consuming applications, and the implications are documented in [3.5. Activity Streams Processing Algorithm][discovery-processing-streams].
 
-The first use case is when third-party aggregators add activities harvested from different institutions, and likely filtered according to some criteria, into a composite stream. For example, a dinosaur aggregator might filter several natural history museum streams for only those manifests that are about dinosaurs.
+The first use case is when third-party aggregators add activities harvested from different institutions, filter those activities or objects, and aggregate them into a composite stream. For example, a dinosaur aggregator might filter several natural history museum streams for only those manifests that are about dinosaurs.
 
-Secondly, resources might become available or unavailable without being created or deleted. This could happen because of changes to their permissions, such as an embargo period after creation, or when resources are temporarily removed in order to be edited as part of some data cleaning or migration process. 
+Secondly, resources might become available or unavailable without being created or deleted. This could happen because of changes to their permissions, such as an embargo period after creation, or when resources are temporarily removed in order to be edited as part of some data cleaning or migration process.
 
-Whenever an "Add" Activity is encountered, it is similar in nature to a "Create" Activity in that it is the first time that resource is mentioned in the stream. It is, therefore, expected that if there is an "Add", then there may not be a "Create" for the same resource occurring before it. Similarly, there will be no more references to the resource after a "Remove" activity, in the same way as for "Delete". 
+Whenever an "Add" Activity is encountered, it is semantically similar to a "Create" Activity in that it is the first time that resource is mentioned in the stream. It is, therefore, expected that if there is an "Add", then there may not be a "Create" for the same resource occurring before it. Similarly, there will be no more references to the resource after a "Remove" or "Delete".
 
 Example Add Activity:
 
-```
-{
-  "type": "Add",
-  "summary": "Added newly discovered manifest to stream",
-  "object": {
-    "id": "https://example.org/iiif/1/manifest",
-    "type": "Manifest"
-  },
-  "target": {
-    "id": "https://example.org/activity/aggregated-changes",
-    "type": "OrderedCollection"
+```json-doc
```

Comment:
Note that I added `json-doc` everywhere, for consistency with our other markdown docs.

Reply:
:+1: